lms/static/js/fixtures/ccx/schedule.json

```diff
@@ -0,0 +1,354 @@
+[{
```

Comment:
Could you run this through a JSON pretty-printer, so that it's easier to read? Here's one that I found with 5 seconds on Google: http://jsonprettyprint.com/